index.js

```diff
@@ -2900,6 +2947,14 @@
 
   return S;
 
+  };
+
+  //  Export two versions of the Sanctuary module: one with type checking;
+  //  one without.
+  var S       = createSanctuary(true);
+  S.unchecked = createSanctuary(false);
+  return S;
+
```

Comment:
> The current solution is similar to using native bindings with node-postgres

That's a useful data point. It's reassuring to see that we independently reached the same conclusion.

> It might be nice to have plain string versions that you can import directly.

I'm reluctant to add complexity to the build step (which is currently non-existent). I'm not sure whether I'm being a pragmatic programmer (having learnt from experience) or whether I'm just being lazy. ;)

> Nothing to do with sanctuary, but you could easily write a babel plugin that creates the appropriate import for you

That's neat! I don't know much about Babel, but it seems very useful for front-end development.